go/teams/box.go

```diff
@@ -0,0 +1,121 @@
+package teams
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+// TeamBox comes from api server team/get endpoint.
+type TeamBox struct {
+	Nonce           string
+	SenderKID       keybase1.KID `json:"sender_kid"`
+	Generation      int
+	Ctext           string
+	PerUserKeySeqno keybase1.Seqno  `json:"per_user_key_seqno"`
+	ReaderKeyMasks  []ReaderKeyMask `json:"reader_key_masks"`
+}
+
+type ReaderKeyMask struct {
+	Application int
```

Comment:
To unmarshal JSON into a struct, Unmarshal matches incoming object keys to the keys used by Marshal (either the struct field name or its tag), preferring an exact match but also accepting a case-insensitive match. Unmarshal will only set exported fields of the struct.